src/test/java/org/fundacionjala/sfdc/stepDefinition/AccountSteps.java

```diff
@@ -0,0 +1,104 @@
+package org.fundacionjala.sfdc.stepDefinition;
+
+import cucumber.api.java.After;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageObjects.SalesForceAccounts;
+import org.fundacionjala.sfdc.pageObjects.SalesForceMainTab;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import java.util.Properties;
+
+/**
+ * AccountSteps.java
+ * Class for steps of account.feature.
+ */
+public class AccountSteps {
+    private WebDriver webDriver;
+    private SalesForceMainTab salesForceMainTab;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "salesForceMainTab" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)